crates/kitsune_p2p/dht/src/region_set/ltcs.rs

```diff
@@ -85,10 +119,11 @@ pub struct RegionSetLtcs<D: RegionDataConstraints = RegionData> {
     #[serde(skip)]
     pub(crate) _region_coords: OnceCell<Vec<RegionCoords>>,
 
-    /// The outer vec corresponds to the spatial segments;
-    /// the inner vecs are the time segments.
+    /// The outermost vec corresponds to arqs in the ArqSet;
+    /// The middle vecs correspond to the spatial segments per arq;
+    /// the innermost vecs are the time segments per arq.
     #[serde(bound(deserialize = "D: serde::de::DeserializeOwned"))]
-    pub data: Vec<Vec<D>>,
+    pub data: Vec<Vec<Vec<D>>>,
 }
```

Comment:
This might be more confusing, but as another option:

```suggestion
    pub data: ArqSetVec<SpatialVec<TimeVec<D>>>,
}

pub type ArqSetVec = Vec;
pub type SpatialVec = Vec;
pub type TimeVec = Vec;
```